packages/rum-core/src/domain/configuration.spec.ts

```diff
@@ -382,4 +443,36 @@ describe('validateAndBuildRumConfiguration', () => {
       ).toBeFalse()
     })
   })
+
+  describe('Serialize RUM configuration', () => {
+    describe('getSelectedTracingHeaders', () => {
+      it('should not return any header type', () => {
+        expect(serializeRumConfiguration(DEFAULT_INIT_CONFIGURATION).selected_tracing_headers).toEqual([])
+      })
+
+      it('should return Datadog header type', () => {
+        const simpleTracingConfig: RumInitConfiguration = {
+          ...DEFAULT_INIT_CONFIGURATION,
+          configureTracingUrls: ['foo'],
+        }
+        expect(serializeRumConfiguration(simpleTracingConfig).selected_tracing_headers).toEqual(['dd'])
+      })
+
+      it('should return Datadog all header types', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
      it('should return all header types', () => {
```

Reply:
👌